src/plugin.rs

```diff
@@ -14,7 +14,7 @@ type RawSymbol<TExportedSymbol> = libloading::os::unix::Symbol<TExportedSymbol>;
 #[cfg(windows)]
 type RawSymbol<TExportedSymbol> = libloading::os::windows::Symbol<TExportedSymbol>;
 
-pub struct Filterer {
+pub(crate) struct Filterer {
```

Comment:
I guess I don't totally understand the visibility rules, I think there was something where even though the module was imported as non-public in `src/lib.rs` (ie `mod whatever;` not `pub mod whatever;`) it complained that a function inside that module that was marked `pub` was taking/returning a `pub(crate)` type?

So I just made everything `pub(crate)` unless it's "truly" `pub` (ie visible to a consumer of this crate)